frontend/src/views/WheresMyGeneV2/common/constants.ts

```diff
@@ -2,6 +2,11 @@ export const GENE_SEARCH_BAR_HEIGHT_PX = 32;
 export const HEATMAP_CONTAINER_ID = "heatmap-container-id";
 export const FMG_GENE_STRENGTH_THRESHOLD = 0.5;
 export const FMG_SPECIFICITY_THRESHOLD = 0.0;
+export const LOADER_HIDE_LABEL_THRESHOLD = 13;
+export const LOADER_POSITION_MULTIPLIER = 19;
+export const LOADER_POSITION_MULTIPLIER_LABEL = 62;
+export const LOADER_POSITION_MULTIPLIER_NO_LABEL = 61;
+export const LOADER_LABEL_TEXT = "Loading Data...";
```

Comment:
super helpful thank you!

Can you help add some comments to explain these constants as well please 🙏 ?

For example, why is the `LOADER_POSITION_MULTIPLIER ` `19`? Why is `LOADER_POSITION_MULTIPLIER_LABEL` `62`, etc.

I'm assuming it has to be something related to geneCount * gene column width, but I can't tell by looking at the constants so far

Thank you for your patience with me here 🙏